api/v1alpha2/zz_generated.conversion.go

```diff
@@ -414,13 +414,13 @@ func Convert_v1alpha1_ClusterList_To_v1alpha2_ClusterList(in *v1alpha1.ClusterLi
 }
 
 func autoConvert_v1alpha2_ClusterSpec_To_v1alpha1_ClusterSpec(in *ClusterSpec, out *v1alpha1.ClusterSpec, s conversion.Scope) error {
-	// WARNING: in.ClusterNetwork requires manual conversion: inconvertible types (*sigs.k8s.io/cluster-api/api/v1alpha2.ClusterNetwork vs sigs.k8s.io/cluster-api/pkg/apis/deprecated/v1alpha1.ClusterNetworkingConfig)
+	// WARNING: in.ClusterNetwork requires manual conversion: inconvertible types (*./api/v1alpha2.ClusterNetwork vs sigs.k8s.io/cluster-api/pkg/apis/deprecated/v1alpha1.ClusterNetworkingConfig)
```

Comment:
Not exactly sure what happened here 🤔

Reply:
Maybe @liztio might know?

Reply:
From best I can tell, this came out of one of the steps from running `make test` (presumably `make generate`)

Reply:
That was my guess as well, I'm just not sure why it would result in different output then when @liztio had initially generated them.

Reply:
Turns out this was a problem with my local setup, I didn't have $GOPATH setup correctly.